.idea/misc.xml

```diff
@@ -0,0 +1,99 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
could you add the .idea folder to the `.gitignore` and delete it?

Reply:
ok, for sure, i am wondering how to delete it because my idea automaticly add it.

Reply:
Maybe it is enough if you add it to the .gitignore, otherwise you can remove it from git in the commant line `git rm -r .idea`